### PR TITLE
Sanatize Title Aswell

### PR DESCRIPTION
**Pull request by 0xEmma**

### CODE REVIEW

File: src/cmds/core/other.py [hunk 1]
```diff
@@ -30,8 +30,11 @@ async def callback(self, interaction: discord.Interaction):
         else:
             title = f"{self.children[0].value}"
         
+
         message_body = self.children[1].value
-        message_body = message_body.replace("@", "[at]") # Slack has no way to disallow @(@everyone calls), so we strip it out and replace it with a safe version
+
+        title = title.replace("@", "[at]") # Slack has no way to disallow @(@everyone calls), so we strip it out and replace it with a safe version
```
Comment (dimoschi) [inline src/cmds/core/other.py:36]: Move the comment above as it's related with both following lines.

File: src/cmds/core/other.py [hunk 2]
```diff
@@ -30,8 +30,11 @@ async def callback(self, interaction: discord.Interaction):
         else:
             title = f"{self.children[0].value}"
         
+
```
Comment (dimoschi) [inline src/cmds/core/other.py:33]: No need for an extra empty line.

**Review by dimoschi — CHANGES_REQUESTED** [review]

**Comment by 0xEmma** [comment]
Requested changes done @dimoschi

**Review by dimoschi — APPROVED** [review]
LGTM

**Pull request merged**